.travis.yml

```diff
@@ -1,261 +1,18 @@
+sudo: false
```

Comment:
Is the intent to use both travis and circle?

Reply:
Yes.  Prior to this PR, travis was the main CI for large image, and circle was only used for testing the mapnik tile source.  With the use of wheels and pytest, both become equally easy.